src/main/java/org/prebid/server/spring/config/ServiceConfiguration.java

```diff
@@ -192,4 +198,14 @@ TimeoutFactory timeoutFactory(Clock clock) {
     BidResponsePostProcessor bidResponsePostProcessor() {
         return BidResponsePostProcessor.noOp();
     }
+
+    @Bean
```

Comment:
Since this service is injected into other components that have scope "prototype" it should have the same scope (due to `HttpClient` sensitivity to event loop it is bound to)